### ISSUE TITLE
Some elements retain default Codeberg style

### ISSUE THREAD

**Issue by hyperreal64** [catppuccin/codeberg#4]
Some elements and boxes retain the default Codeberg theme colors. I took screenshots.

### Watch/Fork/Star buttons:
![Screenshot from 2023-02-28 08-50-27](https://user-images.githubusercontent.com/23226432/221924132-57a7490b-8ca3-48fd-80c7-878ccb0ba7da.png)

### Branch and clone URL buttons:
![Screenshot from 2023-02-28 08-50-09](https://user-images.githubusercontent.com/23226432/221924151-bfebe56a-2e36-43b1-893f-f8c61ff5fedd.png)

### Commit statistics graph:
![Screenshot from 2023-02-28 08-49-45](https://user-images.githubusercontent.com/23226432/221924175-ee9d3757-eaeb-4452-9e8b-ce3778e7b48c.png)

**Comment by justTOBBI** [comment]
Hey, I pushed a new version in the dev branch but I didnt test everything yet, so I didnt merge it yet. Feel free to install the theme manually (open the css and then click on raw) and see if it looks better

**Comment by hyperreal64** [comment]
Yes, it looks better and all the issues from the screenshots I posted are fixed.

**Comment by hyperreal64** [comment]
Sorry, spoke too soon. The element below still has the Codeberg colors.

![Screen Shot 2023-02-28 at 15 11 58](https://user-images.githubusercontent.com/23226432/221981170-8e761c7e-6909-4d0a-b5ff-0387b0123051.png)

After adding these lines to the stylesheet, it looks much better:
```css
     .ui.labeled.button.tooltip.disabled {
            background-color: hsla($accent, 1.0);
        }
        
        .ui.labeled.button.tooltip.disabled, .ui.compact.small.basic.button {
            background-color: hsla($accent, 1.0);
            color: $mantle;
        }
```
![Screen Shot 2023-02-28 at 15 18 19](https://user-images.githubusercontent.com/23226432/221982357-e2025d00-1643-4868-8a5e-7cd3f599ff61.png)

**Comment by justTOBBI** [comment]
Oh right. I forgor that one. I'll look into it soon
Edit: or you can also PR the changes

**Comment by justTOBBI** [comment]
Closed by #5
